aten/src/ATen/Dispatch.h

```diff
@@ -414,9 +414,9 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
   [&] {                                                                     \
```

Comment:
I think we can remove this as well?

Reply:
Won't it be used in https://github.com/pytorch/pytorch/pull/37791?

Reply:
no

Reply:
I removed it